base_multi_company/models/multi_company_abstract.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class MultiCompanyAbstract(models.AbstractModel):
+
+    _name = "multi.company.abstract"
+    _description = "Multi-Company Abstract"
+
+    company_id = fields.Many2one(
+        string="Company",
+        comodel_name="res.company",
+        compute="_compute_company_id",
+        search="_search_company_id",
+    )
+    company_ids = fields.Many2many(
+        string="Companies",
+        comodel_name="res.company",
```

Comment:
@JoanSForgeFlow @JordiBForgeFlow I'm just wondering why the comodel name has been changed during migration to v13 and why not having done migration scripts ? Is there a reason ?

Thanks

Reply:
Ok, didn't see beginning comments